pages/dkp/kaptain/1.3.0/uninstall/index.md

```diff
@@ -0,0 +1,131 @@
+---
+layout: layout.pug
+navigationTitle: Uninstall
+title: Uninstall Kaptain
+menuWeight: 7
+excerpt: Uninstall Kaptain from your cluster
+beta: false
+enterprise: false
+---
+
+<p class="message--warning"><strong>WARNING: </strong>When uninstalling Kaptain, all data, the current state, and configuration of your application will be lost.</p>
+
+## Uninstall Kaptain
+
+Uninstalling Kaptain requires the execution of some manual steps in addition to the regular "KUDO uninstall" command.
+
+1.  Delete all profiles configured with Kaptain. Profiles have dependencies (such as finalizers) associated with Kaptain, and removing Kaptain before removing your profiles will put your cluster in a broken state.
+
+    The following command deletes all notebooks, pipelines and deployed models. Notebook volumes created when deploying notebooks are cleaned up and deleted. **Ensure you have backed up** anything that is not in managed storage (even if model artifacts are stored in your object store, and your notebook code is in version control).
```

Comment:
@cneth there is a way to back up and restore data. However, product and the team have decided not to document this as there is no way to guarantee it will work (at least for this release). Should we explicitly call this out? WDYT?

Reply:
@alembiewski and @chessahoekstra

Reply:
Wait, please help me understand this @chessahoekstra @timcharper @alembiewski 

We don't know how to back up this data?  Or we don't think a backup will actually work?
Why do we bother telling the user to do it if it doesn't work?  And why can't we give them a hint how to do it.

This kind of documentation just creates support problems, and customer satisfaction issues. Customers will read this warning and want to do the thing it instructs them to do - but there's no instructions on how to do it.  Not even a clue...  What am I missing?

If the user can't actually successfully back up this data, we should be honest and tell them that, not try to delude them into thinking it's possible and just frustrating them.